requirements_test.txt

```diff
@@ -0,0 +1,5 @@
+# Mandatory
```

Comment:
just to understand?
what happens if certain dependencies are not isntalled. 
Are the tests marked as skipped or as failed?